tests/PHPUnit/Fixtures/UITestFixture.php

```diff
@@ -49,6 +53,10 @@ public function setUp()
             "idsite = 1"
         );
 
+        // for proper geolocation
+        LocationProvider::setCurrentProvider(LocationProvider\GeoIp\Php::ID);
+        IPAnonymizer::deactivate();
+
```

Comment:
Would it make sense to make a more specific fixture? This change + adding `self::downloadGeoIpDbs()` makes the base fixture more complex (and maybe slower, not sure about that), that's why I'm asking.